scripts/zones/Newton_Movalpolos/IDs.lua

```diff
@@ -22,15 +22,23 @@ zones[xi.zone.NEWTON_MOVALPOLOS] =
         COME_CLOSER             = 7238, -- H0000! C0mE cL0SEr! C0mE cL0SEr! CAn'T TrAdE fr0m S0 fAr AwAy!
         MINING_IS_POSSIBLE_HERE = 7246, -- Mining is possible here if you have <item>.
         CHEST_UNLOCKED          = 7261, -- You unlock the chest!
+        COLLECTOR_SPAWN         = 7270, -- You gently place the <item> on the ground. Suddenly, you hear the clanging sound of armor approaching. It looks like someone is interested in the <item>...
         SHOWMAN_DECLINE         = 7272, -- ... Me no want that. Thing me want not here! It not being here!!!
         SHOWMAN_TRIGGER         = 7273, -- Hey, you there! Muscles nice. You want fight strong one? It cost you. Give me nice item.
         SHOWMAN_ACCEPT          = 7274, -- Fhungaaa!!! The freshyness, the flavoryness! This very nice item! Good luck, then. Try not die. One...two...four...FIIIIIIGHT!!!
         HOMEPOINT_SET           = 7277, -- Home point set!
     },
     mob =
     {
-        MIMIC                = 16826564,
-        BUGBEAR_MATMAN       = 16826570,
+        SWASHSTOX_BEADBLINKER_PH =
+        {
+            [16826507] = 16826517, -- 92.145, 15.500, 66.595
+            [16826514] = 16826517, -- 88.412, 15.421, -19.950
+        },
+        SWASHSTOX_BEADBLINKER    = 16826510,
```

Comment:
Since there is actually 2 copies of this, just like leaping lizzy in south gusta, I would expect him to have a PH for each ID and function similarly - with one of the above to PH entries having this ID for what the spawn rather than it on a separate line. But I've never camped this guy that I can remember - does he work differently?